### ISSUE TITLE
bug:testnet:Wallet get stuck, doesn't show balance if a related staking-pool account do not exists anymore

### ISSUE THREAD

**Issue by luciotato** [near/near-wallet#1670]
**Problem**
Wallet get stuck, doesn't show balance if a related staking-pool account do not exists 
![image](https://user-images.githubusercontent.com/1760905/115253612-ecd01b00-a102-11eb-8172-28ac1d7128c0.png)

```
errors.js:18 Uncaught (in promise) Error: Querying call/devmeta.pool.testnet/get_account_total_balance,Rg2GjYF3S86X7vDePVGzoN6i5vTDgYFDGguMfTQSg failed: Account "devmeta.pool.testnet" doesn't exist.
{
  "error": "Account \"devmeta.pool.testnet\" doesn't exist",
  "logs": [],
  "block_height": 44551649,
  "block_hash": "9kb4M4BExEovJshgaZMzetJG8v1k4Ay5qUrZjBV99gic"
}
    at f.query (json-rpc-provider.js:80)
    at async g.viewFunction (account.js:271)
```

**Expected Behavior**
Ignore non-existing staking-pool and continue.

**Steps to reproduce**

1. testnet: deploy a staking-pool, stake in that pool
2. delete staking-pool account
3. refresh wallet

**Comment by jimmy3dita** [comment]
@luciotato were you able to delete the staking pool with third-party tokens staked into it? Where did the funds go?

**Comment by luciotato** [comment]
`devmeta.pool.testnet` had a test version of the meta-pool. The meta-pool acts as a staking-pool and a NEP-141 at the same time (implements both traits, and it's recognized by the wallet as a staking-pool and a token at the same time)
It does not store and stake tokens itself, it just distributes stakes between several validators. 
I used the the account to test main state migrations, and later deleted the account. https://explorer.testnet.near.org/accounts/devmeta.pool.testnet
It looks like, at some point, I used it with the wallet, and maybe there's a function-call access-key in my account?
I don't know why the wallet is trying to access that `devmeta.pool.testnet`
TBH I'm posting this to make the wallet more resilient, but I don't think this can happen frequently in mainnet.

**Comment by jimmy3dita** [comment]
Can you try logging-in with an "incognito" window, and see if the problem is persistent?

**Comment by luciotato** [comment]
Yes, same problem after recovering the account. Let me try to reproduce with another account

![image](https://user-images.githubusercontent.com/1760905/115788827-8bc27480-a39a-11eb-9160-1d254084786b.png)

**Comment by luciotato** [comment]
I can't reproduce. 
I'll try removing function-call access keys from my account to see if the problem disappears
I'll DM you Stefano

**Comment by jimmy3dita** [comment]
@MaximusHaximus this is material for our "product-readiness" initiative >> test how new releases interact with older local-storage
